backend/src/backend/router.gleam

```diff
@@ -100,7 +101,25 @@ pub fn auth(req: wisp.Request, ctx: Context) -> wisp.Response {
 }
 
 pub fn project(req: wisp.Request, ctx: Context) -> wisp.Response {
-  wisp.response(501)
+  let assert ["api", "v1", "project", ..route] = wisp.path_segments(req)
+
+  case route {
+    [] -> {
+      case req.method {
+        Get -> {
+          wisp.response(501)
+        }
+        Put -> {
+          use params <- get_required_query(req, ["projectid"])
+          let assert [projectid] = params
+
+          project.create_project(projectid, "0", ctx)
+        }
+        _ -> wisp.bad_request()
+      }
+    }
+    _ -> wisp.bad_request()
```

Comment:
I think the api would be a little clearer if it was
GET /project -> list of projects your joined
GET /project/:id -> project details
POST /project/:id BODY=description -> create project
DELETE /project/:id -> delete project
PUT /project/:id?userid -> add userid to project

idk this is kinda verbose, maybe it could be more concise